README.md

```diff
@@ -0,0 +1,37 @@
+# Hathor Wallet Service -- Sync Daemon
+
```

Comment:
It would be nice to have simple commands to install, run a local environment, build, update staging, and deploy. If it gets too big, we can move to separate files.

Reply:
Documented how to run locally, how to build the docker image and how to run using docker

The commit is: bff6a3fc384a9f5630b5a68ce2c9d7ce13cac787

Not sure if I should explain more in depth how to run the fullnode and the wallet-service. What do you think?

Reply:
It seems good enough to me. Thanks!